DOCSP-45863-embedded-verifier-migrations (#557)
mongodb/docs-cluster-to-cluster-sync: source/reference/verification/embedded.txt

of checks on the destination cluster to verify the sync of
supported collections. 

.. note:: 

>    ``mongosync`` reads using :readmode:`primary` read preference, so it 
>    preserves document field order from the source cluster's primary node. The 
>    embedded verifier also checks documents based on the source cluster’s primary 
>    node. Because of this read preference, the verifier cannot verify 
>    discrepancies between the different source cluster nodes. In rare cases, 
>    discrepancies in document field order between the source cluster’s nodes will 
>    cause the embedded verifier to fail the migration, even if ``mongosync`` 
>    copied the documents correctly.

Reviewer: ```suggestion
   ``mongosync`` reads using :readmode:`primary` read preference, so it 
   preserves document field order from the source cluster's primary node. The 
   embedded verifier also checks documents based on the source cluster’s primary 
   node, but at a different time than mongosync reads them. Because of this, in rare cases, 
   discrepancies in document field order between the source cluster’s nodes will 
   cause the embedded verifier to fail the migration, even if ``mongosync`` 
   copied the documents correctly.
```

Reviewer: what do you think about this change @ajayvijayakumar123

Reviewer 2: yup that sounds good. should we also note that discrepancies among cluster nodes themselves are rare?

Reviewer: I think that's already implied in the suggested text

Author: Got it, thanks both. Updated.
